scripts/variantstore/wdl/GvsAoUReblockGvcf.wdl

```diff
@@ -64,11 +64,15 @@ task ReblockAndCopy {
   String has_service_account_file = if (defined(service_account_json)) then 'true' else 'false'
   String gvcf_path = if (defined(service_account_json)) then basename(gvcf) else gvcf
 
-  String dir = if defined(site_id) then (
-      if select_first([site_id]) == "bi" then "gs://prod-genomics-data-broad/" else
-      (if select_first([site_id]) == "bcm" then "gs://prod-genomics-data-baylor/" else
-      (if select_first([site_id]) == "uw" then "gs://prod-genomics-data-northwest/" else "null" )))
-    else ""
+  # janky lower
```

Comment:
please someone tell me a better way to do this

Reply:
I couldn't find a way.  If no one else can, I would just add a comment that this is WDL for `.toLowerCase()`.

Reply:
good call, thanks!